lib/components/form/batch-styled.ts

```diff
@@ -1,35 +1,34 @@
-import {
-  SettingsSelectorPanel,
-  Styled as TripFormClasses
-} from '@opentripplanner/trip-form'
 import styled, { css } from 'styled-components'
 
-import { commonInputCss, modeButtonButtonCss } from './styled'
-import DateTimePreview from './date-time-preview'
+export const buttonPixels = 51
 
-const activeCss = css`
+export const activeCss = css`
   background: #d5d5d5;
```

Comment:
The mode selector gets darker on hover, while this gets lighter. Can we have this go darker instead?

Reply:
Oh I see, it is a Safari thing! Addressed in https://github.com/opentripplanner/otp-react-redux/pull/955/commits/cfd7ee8eb08fc8b774a680fd636b9a23b9d2547d.